common/models/create_environment_request.go

```diff
@@ -41,6 +41,26 @@ func (r CreateEnvironmentRequest) Validate() error {
 		return fmt.Errorf("setting `Scale` is not valid for dynamic environments")
 	}
 
+	if r.EnvironmentType == "" {
+		return fmt.Errorf("EnvironmentType is required, please specify 'static' or 'dynamic'")
+	}
+
+	if r.InstanceType == "" {
```

Comment:
I think we should add a default value for instance type to be dynamic in environment_create.go